media-libs/amdgpu-pro-vulkan/files/vk_pro

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+VK_DRIVER_FILES="/usr/share/vulkan/icd.d/amd_pro_icd32.json:/usr/share/vulkan/icd.d/amd_pro_icd64.json" $*
```

Comment:
okay apparently `$*` doesn't handle well arguments that are quoted (like quoted filenames with spaces)

So it needs to be like this:
```
VK_DRIVER_FILES="/usr/share/vulkan/icd.d/amd_pro_icd32.json:/usr/share/vulkan/icd.d/amd_pro_icd64.json" "$@"
```

Reply:
fixed